src/trading/suggestSlippageBpsFromFee.ts

```diff
@@ -0,0 +1,94 @@
+import { applyPercentage, percentageToBps } from 'src/common/utils/math'
+
+const MIN_SLIPPAGE_BPS = 50
+const MAX_SLIPPAGE_BPS = 10_000 // 100% in BPS
+const ONE_HUNDRED_PERCENT_DECIMAL = 1 // 100 in decimal percentage
+
+const SCALE = 10n ** 18n // 18 decimal places of precision. Used to avoid depending on Big Decimal libraries
```

Comment:
Any particular reason for using 18 decimals precision?
Is it because most tokens have 18 decimals?
How would that work with tokens that have less or more than 18 decimals?

Reply:
No particular reason. Just to get a ton of precision I don't need. 

I added 6 decimals now, which is enough.